composeApp/src/commonMain/kotlin/org/mifos/screen/ApiCallScreen.kt

```diff
@@ -35,59 +35,66 @@ import androidx.compose.ui.graphics.RectangleShape
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import kotlinx.coroutines.launch
 import org.jetbrains.compose.ui.tooling.preview.Preview
-import org.mifos.CallApi
+import org.koin.compose.viewmodel.koinViewModel
 import org.mifos.theme.indigoViolet
 import org.mifos.theme.lightGray
 import org.mifos.theme.mistGray
-import org.mifos.utils.apiParametersValues
+import org.mifos.utils.ApiIdentifier
+import org.mifos.utils.ApiState
+import org.mifos.viewmodel.ApiCallViewModel
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun HomeScreen() {
-    val apiParameters = remember {apiParametersValues()}
+fun apiCallScreen(
+    apiCallViewModel: ApiCallViewModel = koinViewModel()
+) {
+
+    val apiParameters by apiCallViewModel.apiParameters.collectAsState()
+    val listState = rememberLazyListState()
+
+
     Scaffold(
         topBar = {
             TopAppBar(
                 title = {
                     Text(
-                        text = "Mifos Android Architecture",
-                        color = Color.White
+                        text = "Mifos Android Architecture", color = Color.White
```

Comment:
this sample app and screen is for testing only so i don't need to ask you for having text in `string.xml` or to use `MaterialTheme.color` cause i believe these might eventually get removed.
this composeApp is for testing purpsose right?